### PR TITLE
Added a Spell Database.

### PR DESCRIPTION
**Pull request by ghost**
added a prototype Spell Database which has to be checked, trashed and improved upon by L33T :roto2_think:

### CODE REVIEW

**Comment by ghost** [comment]
:+1:

File: Core/Enumerations/CastTypes.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+namespace LeagueSharp.SDK.Core.Enumerations
+{
+    /// <summary>
+    /// Indicates how a spell can be casted
+    /// </summary>
+    public enum CastType
+    {
+        /// <summary>
+        /// The spell can be casted on an enemy champion
```
Comment (zezzy) [inline Core/Enumerations/CastTypes.cs:26]: Excellent work on this line.

File: Core/Enumerations/SpellTags.cs [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        
+        /// <summary>
+        /// The spell can heal
+        /// </summary>
+        Heal,
+
+        /// <summary>
+        /// The spell makes the target enter a stasis state (invulnerable)
+        /// </summary>
+        Stasis,
```
Comment (Cl34r) [inline Core/Enumerations/SpellTags.cs]: I am not sure what level of detail you are going for here so my question might be irrelevant but what about Kayle's R (invulnerable), Fizz's E (untargetable), and Olaf's R (immune to CC)? If you mean Stasis as in Hourglass those three I mentioned are different and don't have a tag.

**Comment by xKurisu** [comment]
visual basic??? ;O

File: Core/Enumerations/CastTypes.cs (hunk 1 above)
Comment (ghost) [inline Core/Enumerations/CastTypes.cs:26]: thank you

**Comment by xKurisu** [comment]
wheres the gui interface :kappa:

**Pull request merged**